InstituteBudget.php

```diff
@@ -640,8 +671,11 @@ public function getDashboardData() {
 		
 		return $data;
 	}
-	
+    
+    
+    //TODO Remove or update once we figure out if this feature is staying
 	public function renderDashboard() {
+        $this->initialize();
```

Comment:
@hdtomp One thing I am thinking about...`initialize()` returns `void` if there is no project ID....what should happen in these other methods if there is no PID? I know not having a project ID is impossible to do on EMs because they are enabled on projects.  if not having a PID is a fail state, then perhaps `initialize()` should throw an exception?

Reply:
You're right that's not a helpful result in the rare case that the pid doesn't exist. I've updated this to log an error and exit instead.